app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,18 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors.order(age: :asc)
+    
+    def average_age(actors)
```

Comment:
This method might have given you trouble because you didn't add an `end` for the `show` method before

What we would like to see is this logic encapsulated in a model method (see commend in the Movie model)